domains/breakertws.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Portfolio by Breaker TWS",
+  "repo": "https://github.com/BreakerTWS/BreakerTWS.github.io",
+  "owner": {
+    "username": "BreakerTWS",
+    "email": "breakingtws@gmail.com"
+  },
+  "record": {
+    "CNAME": "breakertws.github.io",
+    "TXT": "0341066cad9d529001925f19bd4d34"
+  }
```

Comment:
Cannot use CNAME in conjunction with other records and vice versa. See docs for more information: https://docs.is-a.dev